.github/workflows/test_dht_node.yml

```diff
@@ -3,14 +3,12 @@ name: Gradido DHT Node Test CI
 on: push
 
 jobs:
-  # only (but most important) job from this workflow required for pull requests
-  # check results serve as run conditions for all other jobs here 
   files-changed:
     name: Detect File Changes - DHT Node
     runs-on: ubuntu-latest
     outputs:
       dht_node: ${{ steps.changes.outputs.dht_node }}
```

Comment:
here is a dependency on database & mariadb

Reply:
Checking for file changes on database and mariadb Docker files will be handle in a follow up PR.
This PR is about removing unnecessary checks for changes on docker-compose files in build jobs.